preupg/conf.py

```diff
@@ -1,48 +1,5 @@
 # -*- coding: utf-8 -*-
 
-"""
-Valid settings are:
-
-# dir where results of analysis are stored
-result_dir = "/var/tmp/preupgrade"
-
-# xccdf profile
-profile = "xccdf_preupg_profile_default"
-
-# URL of frontend
-frontend = ""
-
-# name of dir with common files
-common_name = "common"
-
-# absolute path to dir with common files
-common_dir = os.path.join(os.path.dirname(__file__),"..", common_name)
-
-# path to file with definitions of common scripts
-common_script = os.path.join(common_dir, "scripts.txt")
-
-# name of file for results
-results = 'results'
-
-# print more data into stdout
-verbose = 1
-
-# skip generation of common files
-skip_common = False
-
-# SCE XML file
-sce_xml = 'sce.xml'
-
-# enable debugging
-debug = False
-
-# id of run in frontend
-id = 123456
-
-# prefix of tag in fccdf files
-xccdf_tag = "xccdf_preupg_rule_"
-"""
-
```

Comment:
This is required for cli.py which uses these values as a default. Otherwise we have to defined the defaults somewhere else.